docs/releases/2.17.md

```diff
@@ -29,3 +29,11 @@ Here are other changes related to the redesign:
 
 ## Upgrade considerations
 
+### Removal of Unidecode & Legacy `clean_name` on `AbstractFormField`
+
+* Unidecode is no longer a required package in Wagtail and the legacy handling of `clean_name` creation on `AbstractFormField` will no longer exist.
```

Comment:
Unidecode has not been a required package since 2.12 (https://github.com/wagtail/wagtail/commit/0e41104670b1d5b2ed507bbd03b8f2d9c3991c6e) - it's only referenced within that one code path for populating clean_name, and anyone relying on that (i.e. because they upgraded straight from <2.10 to 2.12 and thus didn't run the system check within that window of versions) would have had to install unidecode specifically.

Reply:
I'll fix that up. 

I see now where my confusion was - I saw the package listed in  setup.py but did not clearly read it was only in testing dependencies. 

Thanks.